distributed/worker.py

```diff
@@ -287,7 +287,7 @@ class Worker(ServerNode):
     **Volatile State**
 
     These attributes track the progress of tasks that this worker is trying to
-    complete.  In the descriptions below a ``key`` is the name of a task that
```

Comment:
Just and FYI -- some folks prefer to use two spaces between sentences, so you'll see a lot of this throughout the codebase. We don't have a strict convention around this, so there's a mixture of one and two spaces used today. I don't think there's anything wrong with the changes in this PR, just wanted to provide some additional context

Reply:
Thanks for the info, I wasn't aware that this is a convention some folks follow. Good to know!

Reply:
I follow it, but pretty much every other english speaker disagrees with me.  I'll probably keep doing it, but won't get in the way of people changing things :)